{"name": "test_sign_up_invalid_bn_number3", "status": "failed", "statusDetails": {"message": "AssertionError: assert 'ח.פ לא תקין' in 'ח.פ./ע.מ'\n +  where 'ח.פ./ע.מ' = <bound method CommonActions.get_attribute_by_textcontent of <src.utils.commonActions.CommonActions object at 0x000001EE6B05B280>>(('css selector', '#root > div > div.modal_modalWrapper.false.modal_open > div > div > div > div > form > div:nth-child(1) > div:nth-child(2)'))\n +    where <bound method CommonActions.get_attribute_by_textcontent of <src.utils.commonActions.CommonActions object at 0x000001EE6B05B280>> = <src.utils.commonActions.CommonActions object at 0x000001EE6B05B280>.get_attribute_by_textcontent\n +      where <src.utils.commonActions.CommonActions object at 0x000001EE6B05B280> = <tests.edgeTests.testSignUpModal.TestSignUpModal object at 0x000001EE6AC88E20>.common_actions", "trace": "self = <tests.edgeTests.testSignUpModal.TestSignUpModal object at 0x000001EE6AC88E20>\ndriver = <selenium.webdriver.edge.webdriver.WebDriver (session=\"8e2f66c30e85536f6cd4feb423c6f1c5\")>\n\n    @allure.description(\"the test is verify, that user can't sign up with invalid business number\")\n    def test_sign_up_invalid_bn_number3(self, driver):\n        self.welcome_to_modal.choose_coktails()\n        self.home_page.sign_in_click()\n        self.sign_up_modal.sign_up_btn_click()\n        self.sign_up_modal.set_bn_number(\"1\" * 10)\n>       assert \"ח.פ לא תקין\" in self.common_actions.get_attribute_by_textcontent(sign_up_lcs[\"bnnum_div\"])\nE       AssertionError: assert 'ח.פ לא תקין' in 'ח.פ./ע.מ'\nE        +  where 'ח.פ./ע.מ' = <bound method CommonActions.get_attribute_by_textcontent of <src.utils.commonActions.CommonActions object at 0x000001EE6B05B280>>(('css selector', '#root > div > div.modal_modalWrapper.false.modal_open > div > div > div > div > form > div:nth-child(1) > div:nth-child(2)'))\nE        +    where <bound method CommonActions.get_attribute_by_textcontent of <src.utils.commonActions.CommonActions object at 0x000001EE6B05B280>> = <src.utils.commonActions.CommonActions object at 0x000001EE6B05B280>.get_attribute_by_textcontent\nE        +      where <src.utils.commonActions.CommonActions object at 0x000001EE6B05B280> = <tests.edgeTests.testSignUpModal.TestSignUpModal object at 0x000001EE6AC88E20>.common_actions\n\ntests\\edgeTests\\testSignUpModal.py:90: AssertionError"}, "description": "the test is verify, that user can't sign up with invalid business number", "start": 1680536591279, "stop": 1680536591878, "uuid": "dc9ed813-4188-4faf-af6f-e89fa33ef4e3", "historyId": "4f72cb04ec2363ba5f885925295fa8b5", "testCaseId": "a6f39f096e139f945c01d226f8b6dd8f", "fullName": "tests.edgeTests.testSignUpModal.TestSignUpModal#test_sign_up_invalid_bn_number3", "labels": [{"name": "parentSuite", "value": "tests.edgeTests"}, {"name": "suite", "value": "testSignUpModal"}, {"name": "subSuite", "value": "TestSignUpModal"}, {"name": "host", "value": "krot"}, {"name": "thread", "value": "19772-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.edgeTests.testSignUpModal"}]}